src/meta/src/manager/env.rs

```diff
@@ -416,11 +417,25 @@ impl MetaSrvEnv {
                     .await?
                     .map(|c| c.cluster_id.to_string().into())
                     .unwrap();
+                let migrations = Migrator::get_applied_migrations(&sql_meta_store.conn).await?;
+                let mut cluster_first_launch = true;
```

Comment:
@yezizp2012 PTAL for this part, thanks!

Reply:
We should check the status before calling `Migrator::up`. At this point all migrations has already been applied. Please add a helper function to get whether the cluster is first launched, together with some necessary comments.

Reply:
> We should check the status before calling `Migrator::up`. At this point all migrations has already been applied. Please add a helper function to get whether the cluster is first launched, together with some necessary comments.

+1. If we check `get_applied_migrations` here, it will contain "m20230908_072257_init" for sure.

Reply:
Fixed, and since we explicitly specify that `use_new_prefix_strategy` should be passed in, there is no need to make any further judgment for ctl,  right?